_includes/nav.html

```diff
@@ -216,12 +185,11 @@
                 Conference</a></li>
             <li><a target="_blank" rel="noopener noreferrer" href="https://www.confluent.io/community/">Meetups &amp;
                 Events</a></li>
-            <li><a target="_blank" rel="noopener noreferrer" href="https://www.confluent.io/blog/">Blog</a></li>
-            <li><a href="https://developer.confluent.io/podcast/">Podcast</a></li>
             <li><a target="_blank" rel="noopener noreferrer"
                 href="https://www.confluent.io/community/ask-the-community/">Ask the Community</a></li>
             <li><a target="_blank" rel="noopener noreferrer" href="https://www.confluent.io/nominate/">Community
                 Catalysts</a></li>
+            <li><a rel="noopener noreferrer" href="#" style="font-weight: normal">Catalyst spotlight</a></li>
```

Comment:
Same here.. the `href` is just set to a hash.

Reply:
Yes @iPzard , this is how it's there in the contentful.

Reply:
Does this do anything? Why only hash?

Reply:
After talking to @rmoff , this sub-menu has been removed. Thanks.